gtk4/src/auto/shortcut_trigger.rs

```diff
@@ -38,10 +38,10 @@ pub const NONE_SHORTCUT_TRIGGER: Option<&ShortcutTrigger> = None;
 
 pub trait ShortcutTriggerExt: 'static {
     #[doc(alias = "gtk_shortcut_trigger_print")]
-    fn print(&self, string: &mut glib::String);
+    fn print(&self, string: &glib::String);
 
     #[doc(alias = "gtk_shortcut_trigger_print_label")]
-    fn print_label<P: IsA<gdk::Display>>(&self, display: &P, string: &mut glib::String) -> bool;
+    fn print_label<P: IsA<gdk::Display>>(&self, display: &P, string: &glib::String) -> bool;
```

Comment:
What are the differences between print & print_label? also should return `Option<String>` I think